src/main/java/instructionstuff/Instruction.java

```diff
@@ -0,0 +1,288 @@
+package instructionstuff;
+
```

Comment:
I would think it's better to have the package without the word "stuff" as it is redundant